public/javascripts/i18n.js

```
@@ -0,0 +1,1080 @@
+    //     %p     - Meridian indicator (AM  or  PM)
+    //     %P     - Meridian indicator (am  or  pm)
+    //     %S     - Second of the minute (00..60)
+    //     %-S    - Second of the minute (0..60)
+    //     %w     - Day of the week (Sunday is 0, 0..6)
+    //     %y     - Year without a century (00..99)
+    //     %-y    - Year without a century (0..99)
+    //     %Y     - Year with century
+    //     %z/%Z  - Timezone offset (+0545)
+    //
+    I18n.strftime = function (date, format, options) {
+        var options = this.lookup('date', options);
```

Biome (1.9.4)

[error] Shouldn't redeclare 'options'. Consider to delete it or rename it.

'options' is defined here:


(lint/suspicious/noRedeclare)